source/read/distinct.txt

```diff
@@ -0,0 +1,174 @@
+several documents have the same value in the ``borough`` field, each value appears in the
+results only once.
+
+Retrieve Distinct Values Across Specified Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can provide a **query filter** to the ``distinct()`` method to find the distinct
+field values across a subset of documents in a collection. A query filter is an expression
+that specifies search criteria used to match documents in an operation. For more information
+about creating a query filter, see the :ref:`php-specify-query` guide.
+
+The following example retrieves the distinct values of the ``borough`` field for
+all documents that have a ``cuisine`` field value of ``'Italian'``:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/read/distinct.php
+      :start-after: start-distinct-with-query
+      :end-before: end-distinct-with-query
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      "Bronx"
+      "Manhattan"
+      "Queens"
+      "Staten Island"
+
+Modify Distinct Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of the ``distinct()`` method by passing an
+array that specifies option values. The following table describes some
+options you can set to customize the operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``collation`` 
+     - | The collation to use for the operation.
+       | **Type**: ``array|object``
+
+   * - ``maxTimeMS``
+     - | The maximum amount of time in milliseconds that the operation can run.
+       | **Type**: ``integer``
+
+   * - ``comment`` 
+     - | The comment to attach to the operation.
+       | **Type**: any valid BSON type
+
+   * - ``readPreference``
+     - | The read preference to use for the operation. To learn more, see
+         :manual:`Read Preference </core/read-preference/>` in the Server manual.
+       | **Type**: ``MongoDB\Driver\ReadPreference``
+
+The following example retrieves the distinct values of the ``name`` field for
+all documents that have a ``borough`` field value of ``'Bronx'`` and a
+``cuisine`` field value of ``'Pizza'``. It also specifies the ``comment`` field
+in an options array to add a comment to the operation:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/read/distinct.php
+      :start-after: start-distinct-with-comment
+      :end-before: end-distinct-with-comment
+      :language: php
+      :dedent:
+
+   .. output:: 
+      :visible: false
+
+      "$1.25 Pizza"
+      "18 East Gunhill Pizza"
+      "2 Bros"
+      "Aenos Pizza"
+      "Alitalia Pizza Restaurant"
+      "Amici Pizza And Pasta"
+      "Angie'S Cafe Pizza"
+      ...
+
+API Documentation
+-----------------
+
+To learn more about the ``MongoDB\Collection::distinct()`` method, see the following
+API documentation:
+
+- `distinct() <{+api+}/method/MongoDBCollection-distinct/>`__
```

Comment:
S: Make this a sentence since there is only one link here
```suggestion
To learn more about the ``MongoDB\Collection::distinct()`` method, see the
`distinct() API documentation <{+api+}/method/MongoDBCollection-distinct/>`__.
```